packages/react/src/lib/renderComponent.tsx

```diff
@@ -193,9 +184,9 @@ const renderComponent = <P extends {}>(
   // Resolve styles using resolved variables, merge results, allow props.styles to override
   const mergedStyles: ComponentSlotStylesPrepared = mergeComponentStyles(
     theme.componentStyles[displayName],
-    { root: props.design },
-    { root: props.styles },
-    { root: animationCSSProp },
+    withDebugId({ root: props.design }, 'props.design'),
+    withDebugId({ root: props.styles }, 'props.styles'),
+    withDebugId({ root: animationCSSProp }, 'props.animation'),
```

Comment:
Will be also always executed

Reply:
I've added debug check to withDebugId, is that enough?